src/components/Tabs.tsx

```diff
@@ -25,6 +28,15 @@ export interface TabsProps<V extends string, X extends Properties> {
   contentXss?: X;
 }
 
+// Tabs can be rendered as Links (omit "onChange") and we'll use React-Router for matching (omit "selected")/
```

Comment:
Per omitting `onChange` and `selected`, should we omit `value` from `RouteTab`'s extension of `Tab`?

Reply:
No, we'll still want `value`. And the reason why we'd want to use both `path` and `value` is so the do the matching an pass the query variables to the tab panel component... which I'm realizing now that I am not doing. To do this we'll need to call `selectedTab.render()` as part of a `<Route path={path} render={selectedTab.render} />`. 
If we do not do this then the contents of each tab wouldn't have access to the current route params. They'd need to do their own `useRouteMatch` locally vs a `useParams`. Which wouldn't be terrible, but would be different from what we're doing today. Thoughts?

Reply:
Ah, do you mean like this in the BP one:

```
    <div data-testid="tabs_content" css={xss}>
      {tabs.map((tab) => (
        <Route key={tab.value} path={tab.path} render={tab.render} exact />
      ))}
    </div>
```

I guess it's using `tab.value` for the `key` ... but otherwise it's using the `tab.path: string | string[]` for what I think you're describing? 

I.e. it sounds like we still only need the `path`...

Reply:
You're right we don't need `value` in `<Route />` component, but `value` is needed for the `href` of each tab.

Reply:
Oh I gotcha, that makes sense.

Wdyt about omitting value and adding in a dedicated href key?

Reply:
Sure, that makes sense. Definitely a better API.